### PR TITLE
add new section

### PR DESCRIPTION
**Pull request by Vilka-Valeri**
add section green

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -30,4 +30,13 @@ footer {
...
     font-size: 18px;
+}
+.section-green {
+    background: lightseagreen;
+    height: 100vh;
+    color:green;
+    padding: 32px;
+}
+.section-orange h2 {
+    font-size: 18x;
```
Comment (ValerieLeonidova) [inline css/style.css:41]: wrong px

File: index.html [hunk 2]
```diff
@@ -81,6 +81,13 @@ <h2>Orange title</h2>
         <a href="#">Learn more</a>
     </section>
 
+    <section class="section-green">
+        <h2>Orange title</h2>
+        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Quidem architecto deleniti quas numquam quaerat odio ipsa necessitatibus et fugit veniam a, reprehenderit amet placeat illum recusandae voluptatum omnis maxime fuga.</p>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Amet incidunt nisi autem dicta. Odio fugit quisquam eos excepturi natus quod magnam, quos quo! Aut ut possimus nihil repellendus facere. Voluptate?</p>
+        <a href="#">Follow us on Fcebook</a>
```
Comment (ValerieLeonidova) [inline index.html:88]: 'target_blank' was missed

File: css/style.css (hunk 1 above)
Comment (Vilka-Valeri) [reply css/style.css]: fixed

File: index.html (hunk 2 above)
Comment (Vilka-Valeri) [reply index.html]: fixed

**Review by ValerieLeonidova — APPROVED** [review]

**Pull request merged**